### DISCUSSION TITLE
Does a <!DOCTYPE html> tag is a HTML tag?

### DISCUSSION THREAD

**Discussion by olx24** [STRINGLABITSOLUTIONS/Bhojon#230] [Q&A] [answered]
Short answer.

**Comment by montasim** [comment, marked as answer]
No, the <!DOCTYPE html> declaration is not an HTML tag. There are many type of HTML e.g. HTML 4.01 Strict, HTML 4.01 Transitional, HTML 4.01 Frameset, XHTML 1.0 Strict, XHTML 1.0 Transitional, XHTML 1.0 Frameset, XHTML 1.1 etc. So, <!DOCTYPE html> is used to instruct the web browser about the HTML page.
